WASX7209I: Connected to process "dmgr" on node hvyatf10_dmgr using SOAP connector;  The type of process is: DeploymentManager
Running applications: Accueil3PG_EAR, CA3PG-API, CA3PG-HorsSag, CA3PG-Sag, CA3PGBankVisibility, CA3PGTestVisibility, bordereaux-ear, cedicam3pg-gpcc_ae, cedicam3pg-testVisibility, cedicam3pg_ae
Stopped applications: ISAG-ear, SEPA_Generator_RB_2023_war, opfTracker_war
WASX7017E: Exception received while running file "/tmp/nfs/projects/EXE_Echanges_Financiers/SEPA/ANSIBLE/jython/check_app_status.py"; exception information: com.ibm.bsf.BSFException: exception from Jython:
Traceback (most recent call last):
  File "<string>", line 21, in <module>
SystemExit: 1

